velox/functions/sparksql/JsonObjectKeys.h

```diff
@@ -51,10 +51,25 @@ struct JsonObjectKeysFunction {
     }
 
     for (auto field : jsonObject) {
+      if (isFatal(field.error())) {
+        // On-Demand only fully validates the values used and the structure
+        // leading to it.
+        return false;
+      }
       out.add_item().copy_from(std::string_view(field.unescaped_key()));
     }
     return true;
   }
+
+ private:
+  // TODO: After upgrade simdjson v3.12.3, we could replace with simdjson
+  // function relevant simdjson function:
+  // https://github.com/simdjson/simdjson/blob/master/include/simdjson/error-inl.h#L10-L12
+  FOLLY_ALWAYS_INLINE bool isFatal(simdjson::error_code error) noexcept {
```

Comment:
If a member function is defined inside the body of a struct or class, it is implicitly considered inline, so we can drop the `FOLLY_ALWAYS_INLINE`.

Reply:
ok, `FOLLY_ALWAYS_INLINE` removed